VimeoNetworking/Sources/Models/GCS.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  GCS.swift
+//  Vimeo
+//
+//  Created by Nguyen, Van on 11/8/18.
+//  Copyright © 2018 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+public class GCS: VIMModelObject
```

Comment:
Should we add `@objc` for this class?

Reply:
Can we maybe rename that class into `GCStorage`?

Reply:
I would err on the addition of `@objc`. The reason is because GCS must be used only by us. If we support Objective-C compatibility, third-party devs can interpret that they can easily opt into using GCS.

Reply:
Good suggestion. If you don't mind, I'd like to keep this class `GCS`, while renaming the array in `VIMUpload` `GCSStorage`.